### ISSUE TITLE
Sketch not running on Mac 10.12

### ISSUE THREAD

**Issue by deladriere** [processing/processing#4745]
I have installed Processing 3.2.3 on 2 Mac Sierra.
It works fine on one machine
It did work on the other
Now the sketch doesn't start
Exporting as an app works however
I tried renaming, moving the sketch but nothing helps

**Comment by deladriere** [comment]
No sketches run. This is not a programming error
Run button turns green. 
Exported sketches as Application works fine
Here is what the console shows when starting processing:
> AMFI: allowing exception handler for 'Processing' (15952) because the process is not restricted.

Pressing run give this : 

> AMFI: allowing exception handler for 'java' (16224) because the process is not restricted.

**Comment by neborion** [comment]
+1 Have the same issue.

**Comment by returnexitsuccess** [comment]
I have the same issue, except I'm on Windows 10 and have tried with Processing 3.2.3 and 3.2.2 with 32 and 64-bit versions. It was working just fine before so I thought it may have to do with some Java update.

**Comment by neborion** [comment]
I solved it without doing it on purpose. I changed a HDD form my computer… It makes no sense to me.
It wasn't the main HDD, and Processing always run from the main computer's SSD.
Has said Deladriere : "it work on one of his machine". It depend of the hardware of the computer ?!
I'm swimming in mystery…
However, I tried Processing 2 before that and it worked… (mOS 10.12 Sierra)
It can maybe save your life…

Oh wait ! This changed too :
I downgrade processing from 3.2.3 to 3.2.1 
Maybe it's just the reason…, the answer…!

(Previous and new are both 1To / 5400r/min. Previous WesternDigital, New Toshiba)
MacBook Pro 17' Early 2011, i7. OS and Processing 3.2.1 on a SSD.
